data/sources/02_rooms-extended.yaml

```
@@ -1,125 +1,125 @@
+  name: Skriptenverkauf Fachschaft Mathe Physik Informatik Chemie / MPIC
   usage:
     name:
-      de: "Geschäftsraum"
-      en: "Retail space"
-    din_277: "NF4.5"
-    din_277_desc: "Verkaufsräume"
+      de: Geschäftsraum
+      en: Retail space
+    din_277: NF4.5
+    din_277_desc: Verkaufsräume
   props:
     links:
       - text:
-          de: "Öffnungszeiten & Artikel"
-          en: "Opening hours & articles"
-        url: "https://skripten.mpi.fs.tum.de/"
+          de: Öffnungszeiten & Artikel
+          en: Opening hours & articles
+        url: https://skripten.mpi.fs.tum.de/
 
 # === Chemie ===
 "5401.01.101I":
-  name: "5401.01.101I (CIP-Raum)" # Original name is "Rechenzentrum" ...
+  name: CIP-Raum # Original name is "Rechenzentrum" ...
 "5406.01.650C":
-  name: "5406.01.650C (Büro Fachschaft Mathe Physik Informatik Chemie / MPIC)" # Original name is "Büro" ...
+  name: Büro Fachschaft Mathe Physik Informatik Chemie / MPIC # Original name is "Büro" ...
 
 # === Maschinenwesen / MW ===
 "5501.EG.101":
-  name: "5501.EG.101 (Fachschaft Maschinenbau / FSMB)"
+  name: Fachschaft Maschinenbau / FSMB
 "5510.EG.011M":
-  name: "5510.EG.011M (Skriptenverkauf Fachschaft Maschinenbau / FSMB)"
-  short_name: "FSMB Skriptenverkauf"
+  name: Skriptenverkauf Fachschaft Maschinenbau / FSMB
+  short_name: FSMB Skriptenverkauf
   usage:
     name:
-      de: "Geschäftsraum"
-      en: "Retail space"
-    din_277: "NF4.5"
-    din_277_desc: "Verkaufsräume"
+      de: Geschäftsraum
+      en: Retail space
+    din_277: NF4.5
+    din_277_desc: Verkaufsräume
   props:
     links:
       - text:
-          de: "Öffnungszeiten & Artikel"
-          en: "Opening hours & articles"
-        url: "https://fsmb.de/studierende/skriptenverkauf/"
+          de: Öffnungszeiten & Artikel
+          en: Opening hours & articles
+        url: https://fsmb.de/studierende/skriptenverkauf/
 
 "5501.02.150":
-  name: "5501.02.150 (IKOM-Büro)"
+  name: IKOM-Büro
   usage: # previously: Meeting room
     name:
-      de: "Büro"
-      en: "Office"
-    din_277: "NF2.1"
-    din_277_desc: "Büroräume"
+      de: Büro
+      en: Office
+    din_277: NF2.1
+    din_277_desc: Büroräume
 "5504.EG.450":
-  name: "5504.EG.450 (Rechnerhalle)"
+  name: Rechnerhalle
 "5508.02.801":  # ID is probably wrong in TUMonline, but we need to use it for compatibility
-  name: "5508.01.801 (Ernst-Schmidt-Hörsaal)"
+  name: Ernst-Schmidt-Hörsaal
   props:
     comment:
-      de: "Es wird auch 5508.02.801 statt 5508.01.801 verwendet, obwohl der Hörsaal im 1. Stock ist."
-      en: "Also 5508.02.801 instead of 5508.01.801 is being used, even though the lecture hall is in the 1st floor."
+      de: Es wird auch 5508.02.801 statt 5508.01.801 verwendet, obwohl der Hörsaal im 1. Stock ist.
+      en: Also 5508.02.801 instead of 5508.01.801 is being used, even though the lecture hall is in the 1st floor.
```

```suggestion
      de: Dieser Hörsaal ist in TUMonline 5508.02.801 statt 5508.01.801, obwohl der Hörsaal im 1. Stock ist.
      en: This lecture hall is 5508.02.801 in TUMonline instead of 5508.01.801, although the lecture hall is on the 1st floor.
```
Do you think a comment like this could clean this up?

---

I think we can shorten the last sentence to "Aus Konsistenzgründen übernehmen wir diese ID." / "For consistency, we also use this ID."